codespell_lib/data/dictionary.txt

```diff
@@ -18751,6 +18757,8 @@ openess->openness
 openin->opening
 openned->opened
 openning->opening
+openscource->open-source, open source, opensource,
+openscourced->open-sourced, open sourced, opensourced,
```

Comment:
Could also add `scource->source`, `scourced->sourced, scoured,` and `scources->sources` ?

Reply:
Haha, and there's also `scouse` and `scouser`, but they're probably too niche to suggest as replacements :wink: 

But `scourcer->scourer, sorcerer,` might be worth adding? :shrug:

Reply:
All done.